Sources/NIOSSL/NIOSSLHandler.swift

```diff
@@ -391,6 +484,10 @@ public class NIOSSLHandler : ChannelInboundHandler, ChannelOutboundHandler, Remo
             context.fireUserInboundEventTriggered(TLSUserEvent.shutdownCompleted)
 
             switch targetCompleteState {
+            case .outputClosed:
+                // No full channel close here. We would expect users to invoke a full close regardless of
+                // previously completed half closures.
```

Comment:
This comment is out of date now, right?

Reply:
Thanks for pointing that out! I have updated the comment so that it reflects the current state of our half-closure behaviour 😄